org.videolan.VLC.json

```diff
@@ -0,0 +1,867 @@
+        }
+      ]
+    },
+    {
+      "name": "libgpg-error",
+      "config-opts": [
+        "--enable-threads"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.osuosl.org/pub/blfs/conglomeration/libgpg-error/libgpg-error-1.25.tar.bz2",
+          "sha256": "f628f75843433b38b05af248121beb7db5bd54bb2106f384edac39934261320c"
+        }
+      ]
+    },
+    {
+      "name": "libgcrypt",
+      "config-opts": [
+        "--disable-dependency-tracking",
+        "--enable-noexecstack",
+        "--disable-O-flag-munging",
+        "--without-capabilities"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.7.4.tar.bz2",
+          "sha256": "3b67862e2f4711e25c4ce3cc4b48d52a58a3afdcd1d8c6a57f93a1c0ef03e5c6"
+        }
+      ]
+    },
+    {
+      "name": "libidn",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/libidn/libidn-1.33.tar.gz",
+          "sha256": "44a7aab635bb721ceef6beecc4d49dfd19478325e1b47f3196f7d2acc4930e19"
+        }
+      ]
+    },
+    {
+      "name": "zvbi",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/zapping/zvbi/0.2.35/zvbi-0.2.35.tar.bz2",
+          "sha256": "fc883c34111a487c4a783f91b1b2bb5610d8d8e58dcba80c7ab31e67e4765318"
+        }
+      ]
+    },
+    {
+      "name": "libdc1394",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.4/libdc1394-2.2.4.tar.gz",
+          "sha256": "a93689a353c241884a98727128f315ecf9965db70dca710b08af10e5fa0d2e6f"
+        }
+      ]
+    },
+    {
+      "name": "libcddb",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/libcddb/libcddb/1.3.2/libcddb-1.3.2.tar.bz2",
+          "sha256": "35ce0ee1741ea38def304ddfe84a958901413aa829698357f0bee5bb8f0a223b"
+        }
+      ]
+    },
+    {
+      "name": "libvpx",
+      "config-opts": [
+        "--enable-runtime-cpu-detect",
+        "--enable-shared",
+        "--enable-pic",
+        "--disable-examples",
+        "--disable-install-docs",
+        "--disable-install-srcs",
+        "--enable-vp8",
+        "--enable-postproc",
+        "--enable-vp9",
+        "--enable-experimental",
+        "--enable-spatial-resampling"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/webmproject/libvpx/archive/v1.3.0.tar.gz",
+          "sha256": "db72881e1c34e4ae92666847844f35dd2dd431bcf6284500b1b396464f882f64"
+        },
+        {
+          "type": "patch",
+          "path": "libvpx-max_align_t.patch"
+        }
+      ]
+    },
+    {
+      "name": "libcaca",
```

Comment:
This is for outputing to a terminal, Flatpak is more focused on graphical applications so if this is an optional feature perhaps disable it and save a few deps?